source/includes/steps-tutorial-ios-swift-sync.yaml

```
@@ -162,39 +156,18 @@ content: |
      :language: swift
 
 ---
-title: Add Partitions and a Project Member Management Button to the Tasks View
+title: Add Project Member Management Button to the Tasks View
 ref: add-partitions-and-project-member-management-to-tasks
 content: |
-  Next, we need to add partition values for tasks and a button to manage members
-  of a project. Navigate to TaskViewController.swift.
-
-  Because we updated our models to include the new ``partition`` field,
-  we need to pass that partition value whenever we create new tasks. To
-  make that partition value available at all times in the Tasks View,
-  add a property that stores the partition value:
-
-  .. literalinclude:: /tutorial/generated/swift-ios/sync/TasksViewController.codeblock.properties.swift
-     :language: swift
-
-  And we need to set that value at the beginning of ``init``:
-
-  .. literalinclude:: /tutorial/generated/swift-ios/sync/TasksViewController.codeblock.set-partition-value.swift
-     :language: swift
-
-  Update the logic for creating Tasks to set that partition value in
-  ``addButtonDidClick``:
-
-  .. literalinclude:: /tutorial/generated/swift-ios/sync/TasksViewController.codeblock.add-button-did-click.swift
-     :language: swift
-
-  You'll also have to add a new method that checks if a project is owned
-  by the current user, which controls whether or not the user can manage
-  the list of users allowed to access that project:
+  Next, we'll to add menu to manage members of a project that you can open with
```

```suggestion
  Next, we'll need to add a menu to manage members of a project that you can open with
```
Seems like we're missing some words. But this is also awkwardly worded - perhaps we could say instead something like "Add a button that enables the user to open a menu to manage project members"?